test/llvm/lifetime-intrinsics/lifetime-start-class.chpl

```diff
@@ -0,0 +1,18 @@
+// This test is used to verify that llvm.lifetime.start instrinsic is being emitted
+// for variables of a class type inside a procedure
```

Comment:
`owned C` is actually a record in the implementation. Could you make a variant of this test that uses `unmanaged` ?

``` chapel
proc mytest_class() {
  var x: unmanaged C = new unmanaged C();
  delete x;
}
```